internal/deputil/url.go

```diff
@@ -14,11 +14,15 @@
 
 package deputil
 
-import "net/http"
+import (
+	"net/http"
 
-// URLChecker returns url if it's status code is 200, otherwise returns empty string
-func URLChecker(url string) string {
-	resp, err := http.Get(url)
+	"github.com/slackapi/slack-cli/internal/slackdeps"
+)
+
+// URLChecker returns url if its status code is 200, otherwise returns empty string
+func URLChecker(httpClient slackdeps.HTTPClient, url string) string {
+	resp, err := httpClient.Get(url)
```

Comment:
note: A future PR should update this to `.Head`. This will require adding `Head` to our `HTTPClient interface` as well.

Reply:
> I fear rate limits might've been blocking downloads in CI sometimes as well 😓

🧠 Hopefully this reduces some flakiness!